composer.json

```diff
@@ -15,13 +15,15 @@
     "require": {
         "php": "^7.1 || ^8.0",
         "ext-tokenizer": "*",
-        "doctrine/lexer": "1.*"
+        "doctrine/lexer": "1.*",
+        "psr/cache": "^1 || ^2 || ^3"
```

Comment:
maybe `psr/cache-implementation` instead of `psr/cache`?
but does it really depend on it, or is this an optional dep?

Reply:
this 2nd question might be the answer to the 1st, which means `psr/cache` is correct.